tests/fsharp/NUnitHelpers.fs

```diff
@@ -1,17 +1,44 @@
-namespace NUnit.Framework
+namespace Xunit
+
+open Xunit
+open Xunit.Sdk
+open System.Threading
+open System.Globalization
+
+type UseCulture(culture: string, uiCulture: string) =
+
+    inherit BeforeAfterTestAttribute()
+
+    let mutable restore = ignore
+
+    new(culture) = UseCulture(culture, culture)
+    override _.Before _ = 
+        let originalCulture = Thread.CurrentThread.CurrentCulture
+        let originalUiCulture = Thread.CurrentThread.CurrentUICulture
+
+        Thread.CurrentThread.CurrentCulture <- CultureInfo(culture)
+        Thread.CurrentThread.CurrentUICulture <- CultureInfo(uiCulture)
+
+        restore <- fun () ->
+            Thread.CurrentThread.CurrentCulture <- originalCulture
+            Thread.CurrentThread.CurrentUICulture <- originalUiCulture
+
+    override _.After _ = restore ()
 
 module Assert =
 
-    [<assembly: NonParallelizable()>]
+    [<assembly: CollectionBehavior(DisableTestParallelization = true)>]
     do()
 
     let inline fail message = Assert.Fail message
 
     let inline failf fmt = Printf.kprintf fail fmt
 
     let inline areEqual (expected: ^T) (actual: ^T) =
-        Assert.AreEqual(expected, actual)
+        Assert.Equal<^T>(expected, actual)
 
-module StringAssert =
+    let inline contains (expected: string) (actual: string) = Assert.True(actual.Contains(expected))
 
-    let inline contains expected actual = StringAssert.Contains(expected, actual)
+    let inline doesNotThrow (action: unit -> unit) =
```

Comment:
Are you sure this is actually needed? I think in xunit, the conventional way in such cases is to just run the code.
We do this in some cases actually, e.g. [here](https://github.com/dotnet/fsharp/blob/27599c066e81b9497c3e46db18cd291724712e6f/tests/FSharp.Compiler.Service.Tests/FSharpExprPatternsTests.fs#L174).